packages/config/config/devices/0x015f/mh-s511.json

```diff
@@ -0,0 +1,358 @@
+		{
+			"productType": "0x5111",
+			"productId": "0x5103"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
+	"paramInformation": {
+		"2": {
+			"label": "Restore State After Power Failure",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
+		},
+		"3": {
+			"label": "All On / All Off",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 255,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable All Off; Disable All On",
+					"value": 1
+				},
+				{
+					"label": "Enable All On; Disable All Off",
+					"value": 2
+				},
+				{
+					"label": "Enable",
+					"value": 255
+				}
+			]
+		},
+		"4": {
+			"label": "LED Backlight Brightness Level",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 10,
+			"defaultValue": 10,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"5": {
+			"label": "Key Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 6,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Single click to switch on/off",
+					"value": 0
+				},
+				{
+					"label": "When turned on, auto off after X seconds",
+					"value": 1
+				},
+				{
+					"label": "When turned off, auto on after X seconds",
+					"value": 2
+				},
+				{
+					"label": "Hold more than 3 seconds, turns on, when released, turns off",
+					"value": 3
+				},
+				{
+					"label": "Single click to switch on/off, Hold more than 3 seconds will turn on, when released, turns off",
+					"value": 4
+				},
+				{
+					"label": "Momentary switch mode 2: hold the key is on, off once released",
+					"value": 5
+				},
+				{
+					"label": "Hold more than three seconds to turn on, Hold more than three seconds to turn off",
+					"value": 6
+				}
+			]
+		},
+		"6": {
+			"label": "Key On Duration",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"8": {
+			"label": "Basic CC Integration Setting",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Basic Set Received, Key 1 responds.  Basic Get Received, Key 1 sends Basic Report.  Key 1 WILL NOT send unsolicited Basic Report (No Endpoint) to Life Line Association",
```

Comment:
These labels are even worse :D
You could put part of this into the parameter description and reference them instead

Reply:
I agree they are very long (they are taken from the manual).  I've reduced down a bunch, and hopefully all ok.